README.md

```diff
@@ -1 +1,16 @@
-# Portfolio
\ No newline at end of file
+# Porfolio Mobile Design 
+The Project for practise how to make better sofware developer porfolio by using HTML and Css
+
+# Build With
+
+* HTML
+* CSS
+# Mockup
+
+![](Image/Mobile_Design.jpg)
+# Live Demo
+[Live Demo Link](https://github.com/microverseinc/readme-template)
+
+# Authors
+
+* GitHub : [@YvesGanza](https://github.com/YvesGanza)
```

Comment:
#### OPTIONAL

- Please if you have a LinkedIn and Twitter account, add them to your readme 🤓 Your README is looking nice 🔥